### DISCUSSION TITLE
Please Help Me Optimize this Config :'( I5 10400f

### DISCUSSION THREAD

**Discussion by stevenbradley22** [swar/Swar-Chia-Plot-Manager#930] [Q&A]
Here's my PC:
CPU: Inter core I5 10400f
Main: Asrock B460M Steel Legend
Ram: 24GB DDR4 2666 Mhz
SSD: Samsung 980 NVME 1TB x 2
HDD: I have like 10 HDDs 4TB but maybe i will make plots for 2 HDDs at once

- name: miracle
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: Z:\Plotter
    temporary2_directory:
    destination_directory: J:\Plots
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 3500
    max_concurrent: 5
    max_concurrent_with_start_early: ???
    initial_delay_minutes: ???
    stagger_minutes: ???
    max_for_phase_1: ???
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

**Comment by davevandyke** [comment]
looks like a good start...

Run Linux for 10 - 15 % better performance (you can run it off a usb key)

threads: I'd set it to 4 but it might not make much difference
max_concurrent_with_start_early: 7
stagger_minutes: 15 ( or however long it takes for your SSD to transfer the plot to your HDD)
max_for_phase_1: 3

**Reply by stevenbradley22** [reply]
thank you ... what does max_concurrent_with_start_early: 7 means ? can you explain it :D

**Comment by davevandyke** [comment]
My basic understanding is that start early will run additional plots when
in progress plots hit stage 4. Stage 4 is less storage and cpu intensive
than other stages. By starting early you can maximize cpu use.

**Reply by stevenbradley22** [reply]
super ... thank you very much
